library/templates/pharmacy_list_form.php

```diff
@@ -59,94 +59,89 @@
-
-            <div class="col px-0 mx-0">
-                <select class="form-control" name="weno_city" id="weno_city" onchange="cityChanged()"><?php echo xlt("Enter City *"); ?></select>
-                <div class="warn"></div>
-            </div>
-            <span class="ml-1">OR</span>
-            <div class="col">
-                <input class="form-control form-control-sm" name="weno_zipcode" id="weno_zipcode" placeholder="Zipcode">
-            </div>
-            <div>
-                <button type="button" class="btn btn-primary btn-sm" onclick="search()"><?php echo xlt("Search"); ?></button>
-            </div>
+    </div>
+    <div class="row px-0 mx-0">
+        <div class="mr-3 col px-0 mx-0">
+            <select class="form-control form-control-sm" name="weno_state" id="weno_state" onchange="stateChanged()">
+                <option value=""><?php echo xlt("State") . " *"; ?></option>
+                <?php while($row = sqlFetchArray($res)){ ?>
+                    <option value="<?php echo text($row['option_id']); ?>"><?php echo xlt($row['title']); ?></option>
+                <?php   
+                }
+                ?>
+            </select>
         </div>
 
-        <div class="row col mt-2">
-            <div>
-                <select class="form-control form-control-sm" name="form_weno_pharmacy" id="weno_pharmacy" onchange="pharmSelChanged()" >
-                    <option value=""></option>
-                </select>
-            </div>
+        <div class="col px-0 mx-0">
+            <select class="form-control" name="weno_city" id="weno_city" onchange="cityChanged()"><?php echo xlt("Enter City") . " *"; ?></select>
+            <div class="warn"></div>
         </div>
-        
-    </form>
-    <div class="row mt-2 mb-1 contianer col-12">
+        <span class="ml-1">OR</span>
+        <div class="mx-3">
+            <input class="form-control form-control-sm" name="weno_zipcode" id="weno_zipcode" placeholder="Zipcode">
+        </div>
+        <div>
+            <button type="button" class="btn btn-primary btn-sm" onclick="search()"><?php echo xlt("Search"); ?></button>
+        </div>
+    </div>
+    <div class="mt-2">
+        <select class="form-control form-control-sm" name="form_weno_pharmacy" id="weno_pharmacy" onchange="pharmSelChanged()" >
+            <option value=""></option>
+        </select>
+    </div>
+    <div class="mt-2 mb-1">
         <button type="button" class="btn btn-primary btn-sm mr-3" onclick="assignPrimaryPharmacy()"><?php echo xlt("Assign Primary Pharmacy"); ?></button>
         <button type="button" class="btn btn-primary btn-sm" onclick="assignAlternatePharmacy()"><?php echo xlt("Assign Alternate Pharmacy"); ?></button>
-        <button type="button" class="btn btn-light btn-sm ml-3" onclick="resetForm()"><?php echo xlt("Reset"); ?></button>
+        <button type="button" class="btn btn-danger btn-sm ml-3" onclick="resetForm()"><?php echo xlt("Reset"); ?></button>
```

Comment:
Use of `btn-danger` should be reserved for critical events, which Resetting a form does not constitute. Consider `btn-secondary`